app/controllers/UploadController.scala

```diff
@@ -78,7 +78,10 @@ class UploadController(override val authActions: HMACAuthActions, awsConfig: AWS
       projectId = atom.plutoData.flatMap(_.projectId),
       s3Key = CompleteUploadKey(awsConfig.userUploadFolder, id).toString,
       assetVersion = -1,
```

Comment:
I don't think we use this now?

Reply:
assetVersion, you mean?

Reply:
Yeah, I'm working on a different feature branch and I could see it being assigned but never read

Reply:
it's just there 'cos it was - never touched it. Can remove if you like or let it get removed in another update, up to you

Reply:
I'll remove it in my branch. No worries